apps/site/lib/site_web/templates/trip_plan/_fare_calculator.html.eex

```diff
@@ -0,0 +1,61 @@
+
+<div class="h2 m-trip-plan-farecalc__title">Fare Calculator</div>
+
+<div class="m-trip-plan-farecalc__table">
+  <div class="m-trip-plan-farecalc__table-cell m-trip-plan-farecalc__subheader">One-Way Fare</div>
```

Comment:
Not urgent, but thinking about accessibility here...  this should probably all be a `<table>`.

Reply:
`<table>` wasn't quite working but I added roles for the divs instead.